tgui/packages/tgui/interfaces/Orbit.js

```diff
@@ -104,6 +105,9 @@ export const Orbit = (props, context) => {
     return compareString(a[0], b[0]);
   });
 
+
+  let antagonists_title = all_antagonists ? "All Antagonists" : "Ghost-Visible Antagonists"
```

Comment:
linter wants a semi here
```suggestion
  let antagonists_title = all_antagonists ? "All Antagonists" : "Ghost-Visible Antagonists";
```

Reply:
also const